src/pybind/mgr/status/module.py

```diff
@@ -252,7 +252,7 @@ def handle_fs_status(self, cmd):
         return 0, "", output
 
     def handle_osd_status(self, cmd):
-        osd_table = PrettyTable(['id', 'host', 'used', 'avail', 'wr ops', 'wr data', 'rd ops', 'rd data'])
+        osd_table = PrettyTable(['id', 'host', 'used', 'avail', 'wr ops', 'wr data', 'rd ops', 'rd data', 'state'])
```

Comment:
I don't mind adding state, but please put it in a separate commit to the bug fix.  Also, please output the state as something like ",".join(osd['state'])

Reply:
like this?
```c
+----+--------+-------+-------+--------+---------+--------+---------+------------+
| id |  host  |  used | avail | wr ops | wr data | rd ops | rd data |   state    |
+----+--------+-------+-------+--------+---------+--------+---------+------------+
| 0  | ceph14 | 1092M | 9211M |    0   |     0   |    0   |     0   | exists,up  |
| 1  | ceph14 | 1092M | 9211M |    0   |     0   |    0   |     0   | exists,up  |
| 2  | ceph14 | 1092M | 9211M |    0   |     0   |    0   |     0   | exists,up  |
| 3  |        |    0  |    0  |    0   |     0   |    0   |     0   | exists,new |
+----+--------+-------+-------+--------+---------+--------+---------+------------+